tests/core/filter-utils/test_filter_helpers.py

```diff
@@ -308,38 +311,38 @@ def test_check_if_to_block_match(block_number, _type, to_block, expected):
         (TOPICS_B_A_C, FILTER_MATCH_ALL, True),
         (TOPICS_B_C_A, FILTER_MATCH_ALL, True),
         # length 1 matches
-        (TOPICS_EMPTY, FILTER_MATCH_ANY_ONE, False),
-        (TOPICS_ONLY_A, FILTER_MATCH_ANY_ONE, True),
-        (TOPICS_ONLY_B, FILTER_MATCH_ANY_ONE, True),
-        (TOPICS_ONLY_C, FILTER_MATCH_ANY_ONE, True),
-        (TOPICS_EMPTY, FILTER_MATCH_ONLY_A, False),
-        (TOPICS_EMPTY, FILTER_MATCH_ONLY_B, False),
-        (TOPICS_EMPTY, FILTER_MATCH_ONLY_C, False),
-        (TOPICS_ONLY_A, FILTER_MATCH_ONLY_A, True),
-        (TOPICS_ONLY_B, FILTER_MATCH_ONLY_B, True),
-        (TOPICS_ONLY_C, FILTER_MATCH_ONLY_C, True),
-        (TOPICS_ONLY_B, FILTER_MATCH_ONLY_A, False),
-        (TOPICS_ONLY_C, FILTER_MATCH_ONLY_A, False),
-        (TOPICS_ONLY_A, FILTER_MATCH_ONLY_B, False),
-        (TOPICS_ONLY_C, FILTER_MATCH_ONLY_B, False),
-        (TOPICS_ONLY_A, FILTER_MATCH_ONLY_C, False),
-        (TOPICS_ONLY_B, FILTER_MATCH_ONLY_C, False),
-        (TOPICS_A_A, FILTER_MATCH_ONLY_A, True),
-        (TOPICS_A_B, FILTER_MATCH_ONLY_A, True),
-        (TOPICS_A_C, FILTER_MATCH_ONLY_A, True),
-        (TOPICS_A_B_C, FILTER_MATCH_ONLY_A, True),
-        (TOPICS_A_C_B, FILTER_MATCH_ONLY_A, True),
-        (TOPICS_B_A, FILTER_MATCH_ONLY_A, False),
-        (TOPICS_B_C, FILTER_MATCH_ONLY_A, False),
-        (TOPICS_B_A_C, FILTER_MATCH_ONLY_A, False),
-        (TOPICS_B_C_A, FILTER_MATCH_ONLY_A, False),
+        (TOPICS_EMPTY, FILTER_MATCH_ONE_OR_MORE, False),
```

Comment:
This one im not sure on.  Intuitively I would think `(None,)` should match all non-anonymous events, but that is assuming.

Reply:
The parity version makes more sense to me. You can already get the geth like functionally by removing the None.

Reply:
Parity doesn't allow things like matching any one, or any two, like go-ethereum does.  go-ethereum essentially treats None as a place specific wildcard.  On the other hand None feels like a bad choice for a wildcard symbol, and is more intuitive that it provide no actual matching function.

Reply:
Ah, right, I got them switched up. The geth one makes more sense to me, too. :+1: